index.html

```diff
@@ -197,7 +197,19 @@ <h3 class="am-connect">LET'S CONNECT</h3>
                                     <div>Skills</div>
                                     <div><img src="./image/right_arrow.png" alt=""></div>
                             </div>        
-                            
+                            <div class="contact_me" id="6">
+                                <h2>Contact me</h2>
+                                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                                <form action="https://formspree.io/f/mbjwvwle" class="contact-form" method="post">
+                                    <ul class="contact-ul">
+                                        <li><input type="text" id="name" required maxlength="30" placeholder="Full Name"></li>
+                                        <li><input type="email" id="email" required placeholder="Email"></li>
+                                        <li><textarea name="text-box" id="text-box" placeholder="Enter text here" required maxlength="500"></textarea></li>
+                                       <li><div class="get-in-touch"><button  type="submit">Get in Touch</button></div></li>
+                                    </ul>
+                                </form>
+                               <hr class="end-line">
+                            </div>
```

Comment:
### Your Design
![your-fixable-form](https://user-images.githubusercontent.com/93039359/178578487-1766dfc2-5dcb-47cb-a216-1b78c28ba93c.png)

### Figma Design
![figma-template (copy)](https://user-images.githubusercontent.com/93039359/178578518-6f342136-e901-4bf7-b08d-c01386a84eb3.png)

- [ ] Please add the following padding in your form shown by the yellow lines in the Figma image above :+1:
- [ ] Please make changes to the background image according to the Figma template above image :+1:
- [ ] Please make your bar color black and bolder as shown in the Figma template above :blush:

Reply:
Changed accordingly.